ch02_Strategie.md

```diff
@@ -15,17 +14,12 @@ verticaal tussen voorzieningen in verschillende situaties en tussen
 organisaties. Deze doelstelling wordt onderschreven door een breed
 scala aan partijen die deelnemen aan het xxx Kennisplatform, waar de
 ontwikkeling van de standaard zijn oorsprong heeft, en is bestendigd
-door Forum
-
-Standaardisatie en het OverheidsBrede Beleidsoverhed Digitale Overheid
-(OBDO), die xxxx standaard hebben opgenomen op de zogenaamde ‘pas toe
-of leg uit’-lijst met andere standaarden die interoperabiliteit
-bevorderen [<span class="underline">zie ook de basisinformatie van het Forum Standaardisatie</span>](https://www.forumstandaardisatie.nl/basisinformatie).
+door Forum Standaardisatie en het OverheidsBrede Beleidsoverhed
+Digitale Overheid (OBDO), die de Digikoppeling standaard hebben opgenomen op de zogenaamde ‘pas toe of leg uit’-lijst met andere standaarden die
+interoperabiliteit bevorderen [zie ook de basisinformatie van het Forum Standaardisatie](https://www.forumstandaardisatie.nl/basisinformatie).
 
-De toetsingsprocedure voor opname van een standaard op _pas toe of leg
-uit_ lijst bestaat uit de volgende
-
-stappen:
+De toetsingsprocedure voor opname van een standaard op
```

Comment:
suggestie: niet opnemen in beheermodel

Reply:
Gedaan. Procedurebeschrijving verwijderd